modules/assim.sequential/R/sda.enkf_refactored.R

```diff
@@ -209,6 +210,7 @@ sda.enkf <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
     #------------------------------------------- Reading the output
     X_tmp <- vector("list", 2) 
     X <- list()
+    if (control$debug) browser()
```

Comment:
should probably remove this debugging line before final PR merge

Reply:
I'm letting the user here to stop the code at different stages of the run and inspect the internal environment of the SDA function and when they are done with that, they can click on resume and the code will go on like it was never paused. This option can be set by setting `control$debug` to TRUE in sda argument.

Reply:
@araiho This could be an alternative for you instead of running the function line by line.